tailwind.config.ts

```diff
@@ -10,22 +10,25 @@ export default {
     },
     // https://www.figma.com/file/6M2LrpSCcB0thlFDaQAI2J/cx_jod_client?node-id=542%3A6241
     colors: {
-      accent: '#697077',
-      'secondary-1': '#c1c7cd',
-      'secondary-2': '#a2a9b0',
-      'secondary-3': '#878d96',
-      'secondary-4': '#697977',
+      accent: '#006db3',
+      'secondary-1': '#47a7e2',
+      'secondary-2': '#1ab1bb',
+      'secondary-3': '#f08958',
+      'secondary-4': '#d260bb',
+      'secondary-5': '#dcdcdc',
       white: '#ffffff',
       black: '#000000',
       'primary-gray': '#333333',
       'secondary-gray': '#767676',
```

Comment:
It seems that this is also changed in the design tokens:
`#6E6E6E`

Reply:
It's updated now as well